src/components/pagenotfound/pagenotfound.component.html

```diff
@@ -1,9 +1,9 @@
-<mat-card>
-  <mat-card-title>
+<mat-card-title>
     <mat-toolbar color="primary">
       <span>Fehler 404</span>
     </mat-toolbar>
-  </mat-card-title>
+</mat-card-title>
+<mat-card>
```

Comment:
Mat card title should not be outside of mat card

Reply:
So I'll have to change them all to mat-toolbar, right?

Reply:
Yes